### PR TITLE
Bump golangci to v1.38.0

### PR DESCRIPTION
**Pull request by gliptak**

### CODE REVIEW

File: .github/workflows/workflow.yaml [hunk 1]
```diff
@@ -16,17 +16,11 @@ jobs:
...
-        env:
-          GOLANGCILINT_VERSION: 1.35.2
-
-      - name: Make lint
-        run: |
-          make lint
+      - name: Lint
+        uses: golangci/golangci-lint-action@v2
+        with:
+          version: v1.35.2
```
Comment (sozercan) [inline .github/workflows/workflow.yaml:22]: did you wanted to bump this to `v1.38.0` as PR title suggests?

Comment (gliptak) [reply .github/workflows/workflow.yaml]: @sozercan good catch thx

**Comment by gliptak** [comment]
@sozercan anything else you would like to see?

**Review by sozercan — APPROVED** [review]
Thanks for the ping, LGTM

**Review by maxsmythe — APPROVED** [review]
LGTM

**Pull request merged**